faq.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+permalink: /faq/
+title: FAQ
+---
+
+## Are there messaging protocols out there that you feel live up to your security and privacy Standards? Where does Signal fall in?
+
+<!-- @todo do we wanna keep this? its kinda status related -->
```

Comment:
maybe rephrase it as "how does this compare with Signal"

Reply:
@oskarth look at  https://github.com/vacp2p/vacp2p.github.io/pull/13/commits/5fb09da52489a0a689404f50db278516244a9c52, does this make more sense?